File: python_t3_full/Clone_1533.py
def flat_sum(lst) :
	if len(lst) == 0 :
		return 0
	hd, tl = lst [0], lst [1 :]
	if isinstance(hd, list) :
		return flat_sum(hd) + flat_sum(tl)
	elif isinstance(hd, Number) :
		return hd + flat_sum(tl)
	else :
		return flat_sum(tl)


 def flat_sum(lst):
	if not lst:
		return 0
	else:
		hd, tl = lst[0], lst[1:]
		if isinstance(hd, list):
			return flat_sum(hd) + flat_sum(tl)
		elif isinstance(hd, numbers.Number):
			return hd + flat_sum(tl)
		else:
			return flat_sum(tl)

File: c_t3_full/Clone_1533.c
int main () {
    int x, number, fac;
    fac = 1;
    printf ("Enter a number:\n");
    scanf ("%d", & number);
    if (number < 0) {
        printf ("Factorial not defined for negative numbers.\n");
        exit (0);
    }
    for (x = 1; x <= number; x++) {
        if (number >= 0)
            fac = fac * x;
        else
            fac = 1;
    }
    printf ("%d! = %d\n", number, fac);
}


 int main () {
    int x, number, fac = 1;
    printf ("Enter a number:\n");
    scanf ("%d", &number);
    if (number < 0) {
        printf ("Factorial not defined for negative numbers.\n");
        exit (0);
    }
    x = number;
    while (x > 0) {
        fac *= x--;
    }
    printf ("%d! = %d\n", number, fac);
}


